pandas/tests/test_base.py

```diff
@@ -972,6 +972,23 @@ def test_memory_usage(self):
             diff = res_deep - sys.getsizeof(o)
             self.assertTrue(abs(diff) < 100)
 
+    def test_searchsorted(self):
+        # See gh-12238
+        for o in self.objs:
+            # sanity check
+            index = np.searchsorted(o, max(o))
+            self.assertTrue(0 <= index <= len(o))
+
+            if isinstance(o, Index):
```

Comment:
I presume you must be referring to the fact that `sorter` should be `None` in all cases.  See my comment above regarding `Series.searchsorted`.